game-core/src/test/java/games/strategy/triplea/delegate/DiceRollTest.java

```diff
@@ -741,4 +750,61 @@ void testSbrRolls() {
     assertThat(StrategicBombingRaidBattle.getSbrRolls(bomber, british), is(2));
     assertThat(StrategicBombingRaidBattle.getSbrRolls(bomber, british), is(2));
   }
+
+  @Nested
+  @ExtendWith(MockitoExtension.class)
+  class SortAaHighToLowTest {
+
+    @Mock private Unit unit1;
+    @Mock private Unit unit2;
+    @Mock private Unit unit3;
+    @Mock private Unit unit4;
+    @Mock private Unit unit5;
+
+    private final Map<Unit, Tuple<Integer, Integer>> unitPowerAndRollsMap = new HashMap<>();
+    private final List<Unit> units = new ArrayList<>();
+
+    @BeforeEach
+    void setUp() {
+      units.addAll(List.of(unit1, unit2, unit3, unit4, unit5));
+      for (final var unit : units) {
```

Comment:
I think it would help maintainers skip a few steps if this for statement were extracted to a method or a comment is added. Can something be done to make it a bit more direct/obvious what is happening?

Reply:
Initially I had all the setup in this method, but because only parts of it were used in each test, this was against the strict stubbing policy of mockito, that's why I moved the stuff in such a weird way, I think I can fix that